recipe/meta.yaml

```diff
@@ -30,22 +31,20 @@ requirements:
     - {{ compiler('m2w64_cxx') }}  # [win]
     - m2w64-make                   # [win]
     - m2-coreutils                 # [win]
-    - m2-patch  # [win]
-    - patch     # [not win]
   host:
     - python
-    - cmdstan 2.31.0
+    - cmdstan 2.33.1
     - cmdstanpy 1.1.0
     - libboost 1.73
-    - numpy {{ numpy }}
+    - numpy 1.23         # [py<311]
+    - numpy {{ numpy }}  # [py>=311]
```

Comment:
I think that libboost and numpy are not necessary.

Reply:
In host only the following is needed for this package.
```
  - python
  - pip
  - setuptools
  - wheel
  - cmstandpy >=1.0.4
```